### PR TITLE
Update getting-started.md

### PR DESCRIPTION
**Pull request by mihanizator**

### CODE REVIEW

File: getting-started.md [hunk 1]
```diff
@@ -59,3 +59,23 @@ $ exogress spawn --access-key-id=<ACCESS_KEY_ID> --secret-access-key=<SECRET_ACC
 ```
 
 That's it. Now the server on port `3000` is available under the selected domain.
+
+## Advanced
```
Comment (glebpom) [inline getting-started.md]: i wouldn't call it an "Advanced" topic

File: getting-started.md [hunk 2]
```diff
@@ -59,3 +59,23 @@ $ exogress spawn --access-key-id=<ACCESS_KEY_ID> --secret-access-key=<SECRET_ACC
...
+
+### Project-level config
+
+Project-level config simplifies configuration in cases when you have multiple client-level configs and want to set some common behaviour across all of them. Does not support upstreams.
```
Comment (glebpom) [inline getting-started.md]: The main difference is that it works even without any running clients. Lack of upstream is not the only difference. E.g. proxy and static-dir handlers are also disallowed. Project config clearly needs an additional section with more information.

File: getting-started.md [hunk 3]
```diff
@@ -59,3 +59,23 @@ $ exogress spawn --access-key-id=<ACCESS_KEY_ID> --secret-access-key=<SECRET_ACC
...
+
+### Certificates
+
+We use LetsEncrypt to issue and renew TLS certificates for your domains. Make sure the status is `Resolved` before sending the traffic to the domain.
```
Comment (glebpom) [inline getting-started.md]: status should be `Active` or `ready`, depending on where you look at it. Resolved related to DNS

**Review by glebpom — CHANGES_REQUESTED** [review]

File: getting-started.md (hunk 1 above)
Comment (mihanizator) [reply getting-started.md]: just leave in a general "Getting started"? my point was that these items don't require immediate setup

**Pull request merged**